src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/assets/baselines/ArtifactsSizes/centos.8-x64.txt

```diff
@@ -0,0 +1,3849 @@
+./sdk/VERSION/Sdks/Microsoft.NET.Sdk/codestyle/vb/pl/Microsoft.CodeAnalysis.CodeStyle.resources.dll: 31232 bytes
+./sdk/VERSION/Sdks/Microsoft.NET.Sdk/codestyle/vb/pl/Microsoft.CodeAnalysis.VisualBasic.CodeStyle.Fixes.resources.dll: 6144 bytes
+./sdk/VERSION/Sdks/Microsoft.NET.Sdk/codestyle/vb/pl/Microsoft.CodeAnalysis.VisualBasic.CodeStyle.resources.dll: 6144 bytes
+./sdk/VERSION/Sdks/Microsoft.NET.Sdk/codestyle/vb/pt-BR/Microsoft.CodeAnalysis.CodeStyle.Fixes.resources.dll: 12288 bytes
+./sdk/VERSION/Sdks/Microsoft.NET.Sdk/codestyle/vb/pt-BR/Microsoft.CodeAnalysis.CodeStyle.resources.dll: 30720 bytes
+./sdk/VERSION/Sdks/Microsoft.NET.Sdk/codestyle/vb/pt-BR/Microsoft.CodeAnalysis.VisualBasic.CodeStyle.Fixes.resources.dll: 6144 bytes
+./sdk/VERSION/Sdks/Microsoft.NET.Sdk/codestyle/vb/pt-BR/Microsoft.CodeAnalysis.VisualBasic.CodeStyle.resources.dll: 6144 bytes
+./sdk/VERSION/Sdks/Microsoft.NET.Sdk/codestyle/vb/ru/Microsoft.CodeAnalysis.CodeStyle.Fixes.resources.dll: 13824 bytes
+./sdk/VERSION/Sdks/Microsoft.NET.Sdk/codestyle/vb/ru/Microsoft.CodeAnalysis.CodeStyle.resources.dll: 38400 bytes
+./sdk/VERSION/Sdks/Microsoft.NET.Sdk/codestyle/vb/ru/Microsoft.CodeAnalysis.VisualBasic.CodeStyle.Fixes.resources.dll: 6144 bytes
+./sdk/VERSION/Sdks/Microsoft.NET.Sdk/codestyle/vb/ru/Microsoft.CodeAnalysis.VisualBasic.CodeStyle.resources.dll: 6144 bytes
+./sdk/VERSION/Sdks/Microsoft.NET.Sdk/codestyle/vb/tr/Microsoft.CodeAnalysis.CodeStyle.Fixes.resources.dll: 11776 bytes
+./sdk/VERSION/Sdks/Microsoft.NET.Sdk/codestyle/vb/tr/Microsoft.CodeAnalysis.CodeStyle.resources.dll: 30208 bytes
+./sdk/VERSION/Sdks/Microsoft.NET.Sdk/codestyle/vb/tr/Microsoft.CodeAnalysis.VisualBasic.CodeStyle.Fixes.resources.dll: 6144 bytes
+./sdk/VERSION/Sdks/Microsoft.NET.Sdk/codestyle/vb/tr/Microsoft.CodeAnalysis.VisualBasic.CodeStyle.resources.dll: 6144 bytes
+./sdk/VERSION/Sdks/Microsoft.NET.Sdk/codestyle/vb/zh-Hans/Microsoft.CodeAnalysis.CodeStyle.Fixes.resources.dll: 11776 bytes
+./sdk/VERSION/Sdks/Microsoft.NET.Sdk/codestyle/vb/zh-Hans/Microsoft.CodeAnalysis.CodeStyle.resources.dll: 28160 bytes
+./sdk/VERSION/Sdks/Microsoft.NET.Sdk/codestyle/vb/zh-Hans/Microsoft.CodeAnalysis.VisualBasic.CodeStyle.Fixes.resources.dll: 6144 bytes
+./sdk/VERSION/Sdks/Microsoft.NET.Sdk/codestyle/vb/zh-Hans/Microsoft.CodeAnalysis.VisualBasic.CodeStyle.resources.dll: 5632 bytes
+./sdk/VERSION/Sdks/Microsoft.NET.Sdk/codestyle/vb/zh-Hant/Microsoft.CodeAnalysis.CodeStyle.Fixes.resources.dll: 11776 bytes
+./sdk/VERSION/Sdks/Microsoft.NET.Sdk/codestyle/vb/zh-Hant/Microsoft.CodeAnalysis.CodeStyle.resources.dll: 28160 bytes
+./sdk/VERSION/Sdks/Microsoft.NET.Sdk/codestyle/vb/zh-Hant/Microsoft.CodeAnalysis.VisualBasic.CodeStyle.Fixes.resources.dll: 6144 bytes
+./sdk/VERSION/Sdks/Microsoft.NET.Sdk/codestyle/vb/zh-Hant/Microsoft.CodeAnalysis.VisualBasic.CodeStyle.resources.dll: 5632 bytes
+./sdk/VERSION/Sdks/Microsoft.NET.Sdk/Sdk/Sdk.AfterCommon.ta
```

Comment:
These version numbers are going to have to be abstracted away otherwise the baseline will differ frequently.